src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -32,46 +30,35 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
     public class ThemeService : IThemeService
     {
         private readonly ContentDbContext _contentDbContext;
-        private readonly StatisticsDbContext _statisticsDbContext;
+        private readonly PublicDataDbContext _publicDataDbContext;
```

Comment:
I *think* that adding this means that creating a ContentDb migration now fails - although it's possible I've got this wrong:

```
An error occurred while accessing the Microsoft.Extensions.Hosting services. Continuing without the application service provider. Error: Some services are not able to be constructed (Error while validating the service descriptor 'ServiceType: GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.ThemeController Lifetime: Transient ImplementationType: GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.ThemeController': Unable to resolve service for type 'GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database.PublicDataDbContext' while attempting to activate 'GovUk.Education.ExploreEducationStatistics.Admin.Services.ThemeService'.) (Error while validating the service descriptor 'ServiceType: GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.IThemeService Lifetime: Transient ImplementationType: GovUk.Education.ExploreEducationStatistics.Admin.Services.ThemeService': Unable to resolve service for type 'GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database.PublicDataDbContext' while attempting to activate 'GovUk.Education.ExploreEducationStatistics.Admin.Services.ThemeService'.)
```

Reply:
Hmm, seems to work OK for me:

Running:

```bash
cd src/GovUk.Education.ExploreEducationStatistics.Admin
dotnet ef migrations add EES1234_MigrationNameHere --context ContentDbContext --output-dir Migrations/ContentMigrations -v
```

I get:

![image](https://github.com/user-attachments/assets/5763d1cd-f173-4909-b307-4178fc9b712b)

Reply:
Fixed